src/osut/osut.py

```diff
@@ -3350,12 +3353,10 @@ def isPointAlongSegment(p0=None, sg=[]) -> bool:
 
     if not isinstance(p0, cl1):
         return oslg.mismatch("point", p0, cl1, mth, CN.DBG, False)
-    if not isSegment(sg):
-        return oslg.mismatch("segment", sg, cl2, mth, CN.DBG, False)
-
+    if not isSegment(sg): return False
```

Comment:
Lowering the log volume: `isPointAlongSegment` calls `isSegment`, which in turn calls `p3Dv`. The latter will log an inadmissible object.